src/main/java/au/id/micolous/metrodroid/card/classic/ClassicCard.java

```diff
@@ -73,8 +77,16 @@
     public static final byte[] PREAMBLE_KEY = {(byte) 0x00, (byte) 0x00, (byte) 0x00, (byte) 0x00, (byte) 0x00,
             (byte) 0x00};
 
+    public static final byte[] TROIKA_SECTOR_8_KEY = {
+            (byte) 0xa7, (byte) 0x3f, (byte) 0x5d, (byte) 0xc1, (byte) 0xd3, (byte) 0x33};
```

Comment:
This one looks like it's in some public key lists, but I'll modify this to use the same "hashing" setup that is done for the Smartrider cards, because I don't want to publish a transit operator's keys with Metrodroid.

I should also document this better, and refactor it out of the Smartrider module.